src/components/common/course-enrollments/course-cards/InProgressCourseCard.jsx

```diff
@@ -26,8 +33,32 @@ const InProgressCourseCard = (props) => {
     return false;
   });
 
+  const getDropdownMenuItems = () => {
+    const isMoveToCompleteEnabled = isFeatureEnabled('move_to_completed');
+    const { pageContext: { pageType } } = useContext(LayoutContext);
+    if (!isMoveToCompleteEnabled || pageType !== 'pages.EnterprisePage') {
```

Comment:
Checking for `pageType` may be a sign that we need to change the way we are reusing code here. If would be a shame to litter the code with logic checks like this.

Reply:
Yeah, I was thinking the same thing. Maybe we can chat a bit about an approach that avoids these logic checks going forward?

Reply:
One way would be to pass the dropdown menu items as props down the component hierarchy from the enterprise-specific parent component.